### PR TITLE
Snake improve

### PR DESCRIPTION
**Pull request by bongiovimatthew**

### CODE REVIEW

File: Environment/JigsawPuzzle/puzzleEnvironment.py [hunk 1]
```diff
@@ -33,9 +34,12 @@ def __init__(self):
         self.debugMode = False
         self.action_space = ActionSpace(range(self.MAX_ACTIONS_NUM))
 
+        img_folder = Path("images/")
+        puzzle_img_path = img_folder / "rainier_small.jpg"
```
Comment (gupta158) [inline Environment/JigsawPuzzle/puzzleEnvironment.py:38]: woahhh

File: Environment/Snake/snake.py [hunk 2]
```diff
@@ -21,8 +20,8 @@ def place(self, player):
         counter = 0
         touchesSnake = True
         while counter < 1000000 and touchesSnake:
-            self.x = randint(1, 17) * 44
-            self.y = randint(1, 12) * 44
+            self.x = randint(0, (800 / 50) - 1) * self.step
```
Comment (gupta158) [inline Environment/Snake/snake.py:23]: can this happen to be put back on a block where the snake is on already? i dont remember if snake is supposed to account fo rthat

**Review by gupta158 — CHANGES_REQUESTED** [review]
Should we account for the apple on snake logic below?

File: Environment/Snake/snake.py (hunk 2 above)
Comment (bongiovimatthew) [reply Environment/Snake/snake.py:23]: The "touchesSnake" logic keeps it from being placed on the snake (within the 1000000 max attempts counter)

**Comment by bongiovimatthew** [comment]
Yeah, I think if I understand your comment correctly, we are already handling the case where we avoid putting the apple someplace the snake already occupies

**Review by gupta158 — APPROVED** [review]

**Pull request merged**